static/js/main.js

```diff
@@ -5,11 +5,40 @@ const body = document.querySelector('body');
 let maxPagesReached = false;
 let loadingData = false;
 let selectedCountry;
-const darkModeIsOn = localStorage.getItem('dark-mode') === 'enabled';
 
-localStorage.getItem('dark-mode') === null ?
-    localStorage.setItem('dark-mode', 'disabled') :
-    body.classList.toggle('dark-mode', localStorage.getItem('dark-mode') !== 'disabled');
+const setCookie = (cookieName, cookieValue, expirationDays) => {
+    const date = new Date();
+    date.setTime(date.getTime() + (expirationDays * 24 * 60 * 60 * 1000)); // Convert expiration days to miliseconds
+
+    const expires = `expires=${date.toUTCString()}`;
+    document.cookie = `${cookieName}=${cookieValue};${expires};path=/`
+}
+
+const getCookies = () => {
```

Comment:
Lijkt me ook niet nodig om cookie getter and setter zo complex, te maken hiervoor. Probeer dit voor jezelf te versimpelen.